Tests/function/tests/Update-VSTeamIteration.Tests.ps1

```diff
@@ -0,0 +1,49 @@
+Set-StrictMode -Version Latest
+
+Describe 'VSTeamIteration' {
+   BeforeAll {
+      . "$PSScriptRoot\_testInitialize.ps1" $PSCommandPath      
+      . "$baseFolder/Source/Public/Update-VSTeamClassificationNode"
+
+      ## Arrange
+      Mock _getInstance { return 'https://dev.azure.com/test' }
+      Mock Invoke-RestMethod { Open-SampleFile 'classificationNodeResult.json' }
+      Mock _getApiVersion { return '5.0-unitTests' } -ParameterFilter { $Service -eq 'Core' }
+   }
+
+   Context 'Update-VSTeamIteration' {
+      It 'iteration should return Nodes' {
```

Comment:
```suggestion
      It 'iteration should update Nodes' {
```

unless I am understanding it wrong it should be update, since you are not checking the returned object.